app/src/main/java/firebasebarcelona/firepadel/ui/courts/view/CourtListFragment.java

```diff
@@ -141,7 +141,7 @@ public void onAuthStateChanged(@NonNull FirebaseAuth auth) {
         if (user != null) {
           saveLocalPlayer(user);
         } else {
-          Toast.makeText(getActivity(), "Auth fail", Toast.LENGTH_SHORT).show();
+          //TODO show some kind of error
```

Comment:
Well, it's a todo, so we have to do it. We don't need any methodology, and I don't have the time to do this design right now. So I prefer to delete a bad feedback from de app and put a TODO